src/python/grpcio/grpc/_cython/_cygrpc/aio/server.pyx.pxi

```diff
@@ -197,21 +197,41 @@ cdef class _ServicerContext:
     def set_trailing_metadata(self, object metadata):
```

Comment:
Please consider add a test case to access these values in interceptor (as your use case) in https://github.com/grpc/grpc/blob/master/src/python/grpcio_tests/tests_aio/unit/server_interceptor_test.py, to guard this change.